benchmarks/media-streaming/client/files/docker-entrypoint.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 set -e
 
+# $1: the IP of the server
+# $2: the number of httperf clients
+# $3: the total number of sessions
+# $4: the rate (sessions per seconds)
```

Comment:
If we are going to make this change to the benchmark, we should update all of the documentation to explain what these parameters are, and how the user can adjust them.

Reply:
The doc file provides information about these input values.

Reply:
Good. However, their explanation does not really match the graphs we have posted on the wiki. We have the rate on the x axis, but this only dictates the "ramp up" rate on the server. The total constant load is the number of sessions, which is what should be on the x-axis?